FASTN.ftd

```diff
@@ -17,6 +17,18 @@ provided-via: ui.fifthtry.com/ds
 -- fastn.dependency: fastn-community.github.io/dark-flame-cs
 -- fastn.dependency: fastn-community.github.io/svg-icons
 -- fastn.dependency: fastn-community.github.io/code-block
+-- fastn.dependency: meenukumari28.github.io/linktree-template
+-- fastn.dependency: meenukumari28.github.io/linktree-template-2
+-- fastn.dependency: meenukumari28.github.io/linktree-template-3
+-- fastn.dependency: meenukumari28.github.io/linktree-template-4
+-- fastn.dependency: meenukumari28.github.io/coming-soon-1
+-- fastn.dependency: meenukumari28.github.io/coming-soon-2
+-- fastn.dependency: meenukumari28.github.io/coming-soon-3
+-- fastn.dependency: meenukumari28.github.io/coming-soon-4
+-- fastn.dependency: meenukumari28.github.io/coming-soon-5
+-- fastn.dependency: meenukumari28.github.io/coming-soon-6
+-- fastn.dependency: meenukumari28.github.io/coming-soon-7
+-- fastn.dependency: meenukumari28.github.io/coming-soon-8
```

Comment:
@MeenuKumari28 we have to move all these to fastn-community github org. Do you have permission to create a new repo in our org, else I will do it. Also need better names than coming-soon-1 etc. coming-soon-ross, coming-soon-phoebe etc you can do. And similarly charlie, alan, jake, rose for the linktree ones.

Reply:
Also these files should not exist in this package: `assets/templates/coming-soon-1.PNG` You should put them in say `-- import: meenukumari28.github.io/coming-soon-8/assets as coming-soon-8-assets`, and use it as `src: $coming-soon-8-assets.files.preview.png`.